pkgs/applications/misc/volnoti/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub
+, pkgconfig, dbus, gdk_pixbuf, glib, libX11, gtk2
+, dbus_glib, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  name = "volnoti-${version}";
+  version = "git-2019-09-26";
```

Comment:
Please change name to `volnoti-unstable-${version}` and version to `2013-09-23`. See about [package naming](https://nixos.org/nixpkgs/manual/#sec-package-naming) in the Nixpkgs manual.